sqlalchemy_bigquery/test.py

```diff
@@ -0,0 +1,6 @@
+from sqlalchemy.sql.expression import cast
```

Comment:
What is the purpose of the code in this file?

Reply:
I am not sure, @nayaknishant do you know, I assume you added this code

Reply:
Not sure, Kira I think you added it in this commit https://github.com/googleapis/python-bigquery-sqlalchemy/commit/5503877901f86a7a52cd1057c13ef8fa5ba5c575

Reply:
It completely slipped my mind but I did add this code to test something, will remove this file in the next commit, thanks all

Reply:
of course, thanks Kira!